src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Android/StorePal.cs

```diff
@@ -44,7 +44,47 @@ public static IExportPal LinkFromCertificateCollection(X509Certificate2Collectio
 
         public static IStorePal FromSystemStore(string storeName, StoreLocation storeLocation, OpenFlags openFlags)
         {
-            throw new NotImplementedException(nameof(FromSystemStore));
+            bool isReadWrite = (openFlags & OpenFlags.ReadWrite) == OpenFlags.ReadWrite;
+            if (isReadWrite && storeLocation == StoreLocation.LocalMachine)
+            {
+                // All LocalMachine stores are read-only
```

Comment:
On Linux it's because we don't actually have a good way to update the stores (on some distros what we see is more or less authoritative, on some it's not).  Can a user create new system entries in the KeyStores? If so, they don't have to be read-only.

Reply:
No, the system entries are entirely locked down from an app's perspective.